Source/Libraries/GSF.PhasorProtocols/UI/DataModels/Device.cs

```diff
@@ -314,7 +314,7 @@ public int AccessID
         /// Gets or sets <see cref="Device"/> Longitude.
         /// </summary>
         // Because of database design, no validation attributes are applied.        
-        [RegularExpression(@"^[-]?([0-9]{1,3})?([.][0-9]{1,6})?$", ErrorMessage = "Invalid value. Please provide value in decimal(9,6) format.")]
+        [RegularExpression(@"^[-]?([0-9]{1,3})?([.,][0-9]{1,6})?$", ErrorMessage = "Invalid value. Please provide value in decimal(9,6) format.")]
```

Comment:
I am assuming `decimal(9,6)` format is `###./,######`? Can we rephrase that to make this clear in the UI?

Reply:
It comes from how precision and scale for a data type are specified in SQL.
https://learn.microsoft.com/en-us/sql/t-sql/data-types/decimal-and-numeric-transact-sql#arguments

Perhaps it was described that way to be precise, because it's actually the data type we use in the database.

https://github.com/GridProtectionAlliance/gsf/blob/b4a75117714335053892e42c8caf1aaaa9297da4/Source/Libraries/GSF.TimeSeries/Data/SQL%20Server/GSFSchema.sql#L326-L327

But I can understand why this isn't really a great way to describe the syntax to the user. Perhaps the most intuitive way to describe it would be between -1000 and 1000 with up to six decimal places. Though it might be better to specify -180 to 180 for longitude and -90 to 90 for latitude, even if that doesn't really match the regex or the data type.